include/pinocchio/spatial/inertia.hpp

```diff
@@ -889,6 +960,366 @@ namespace pinocchio
 
   }; // class InertiaTpl
 
+  /**
+   * @brief A structure representing a pseudo inertia matrix.
+   *
+   * References:
+   * - Wensing, Patrick M., Sangbae Kim, and Jean-Jacques E. Slotine. "Linear matrix inequalities
+   * for physically consistent inertial parameter identification: A statistical perspective on the
+   * mass distribution." IEEE Robotics and Automation Letters 3.1 (2017): 60-67.
+   */
+  template<typename Scalar, int Options>
+  struct PseudoInertiaTpl
+  {
+    typedef Eigen::Matrix<Scalar, 4, 4, Options> Matrix4;
+    typedef Eigen::Matrix<Scalar, 3, 1, Options> Vector3;
+    typedef Eigen::Matrix<Scalar, 3, 3, Options> Matrix3;
+    typedef Eigen::Matrix<Scalar, 10, 1, Options> Vector10;
+    typedef LogCholeskyParametersTpl<Scalar, Options> LogCholeskyParameters;
+
+    Scalar mass;    ///< Mass of the pseudo inertia
+    Vector3 h;      ///< Vector part of the pseudo inertia
+    Matrix3 Sigma;  ///< 3x3 matrix part of the pseudo inertia
+    Matrix4 matrix; ///< Full 4x4 pseudo inertia matrix
```

Comment:
matrix is redundant with mass, h and sigma.
This can lead to some error: if an user modify mass, matrix is no longer valid.

I see two options:
- Only storing matrix and provide getter to obtain mass, h and sigma
- Only store mass, h, sigma and don't store matrix

I prefer the second option: less code, and the matrix is only used in the `disp_impl` function.